dockerfiles/dockerfile.inttests

```diff
@@ -0,0 +1,11 @@
+ARG baseVersion
+ARG dockerRepository
+FROM ${dockerRepository}/galasadev/galasa-base:${baseVersion}

```

Comment:
Please point this at the new base image location in GHCR which is at: `FROM ${dockerRepository}/galasa-dev/base-image:${baseVersion}`

Reply:
done